haystack/document_stores/sql.py

```diff
@@ -51,8 +49,24 @@ class MetaDocumentORM(ORMBase):
         nullable=False,
         index=True
     )
+    document = relationship("DocumentORM", back_populates="meta")
+
+
+class LabelORM(ORMBase):
+    __tablename__ = "label"
 
-    documents = relationship("DocumentORM", back_populates="meta")
+    # Composite PK with id allows the same label in many indices
+    index = Column(String(100), nullable=False,primary_key=True)
+    
+    query = Column(Text, nullable=False)
+    answer = Column(JSON, nullable=True)
```

Comment:
JSON doesn't work in older MYSQL and postgres. Also, there are a few issues with it
https://amercader.net/blog/beware-of-json-fields-in-sqlalchemy/

Reply:
I'll keep it in mind! This in fact was not an addition of mine, I just moved `LabelORM`

Reply:
Yeah just added so we can add in a comment but check about it in the future.